### DISCUSSION TITLE
How can i make the style follow the directories?

### DISCUSSION THREAD

**Discussion by mayhmemo** [sixem/ivfi-php#22] [Questions] [answered]
I don't know php and i have a problem. The indexer.php works very well but the style (in the css) don't follow for the rest directories.
![public](https://user-images.githubusercontent.com/84481401/118896758-56e50700-b8df-11eb-8a5c-a3b22a2e6be4.png)
When i click at "/archives" for example it give me a normal index page.
![archives](https://user-images.githubusercontent.com/84481401/118896757-551b4380-b8df-11eb-919a-a2f281710cff.png)
I'm probably being just stupid, but i really don't know how to fix it.

**Comment by sixem** [comment, marked as answer]
What are you using to set it up? Apache? And if so, `.htaccess` or configuration files? Or something else?

It's been a while since I've used Apache, but if you are using something like `.htaccess`, I think it can cause conflicts if you, for example, have another `.htaccess` file in `/archives` that hasn't been set up with the Indexer.

The problem here isn't just the style, the Indexer is just not being applied at all for `/archives`, so there are some configuration issues here.

**Reply by mayhmemo** [reply]
Yes, i'm using Apache with .htaccess, but i don't have a second .htaccess file in" /archives".
Here is some prints of my directory and the print of .htaccess text.
![public-html](https://user-images.githubusercontent.com/84481401/119007476-a4a55200-b967-11eb-9c84-80af28f5a028.png)
![archivess](https://user-images.githubusercontent.com/84481401/119007484-a5d67f00-b967-11eb-9d40-a040a27facdb.png)
![htaccess](https://user-images.githubusercontent.com/84481401/119007633-ca325b80-b967-11eb-9a1d-9f6b154ed226.png)

**Reply by sixem** [reply]
Is that a space between `.index.php/` and `indexer.php`? Try to correct it to `.index.php /indexer.php` instead.

**Reply by mayhmemo** [reply]
It worked! Thanks a lot!!

**Reply by sixem** [reply]
No problem!
